Snake/GameBoard.java

```diff
@@ -224,30 +285,32 @@ private void paintFood (Graphics2D g) {
                 Properties.SQUARE_SIZE - 2, corner, corner);
     }
 
-    @Override
-    public String toString () {
+	@Override
+	public String toString() {
```

Comment:
This isn't a big deal but none of these lines should've been changed.
Be very careful about this in the future.